lib/language_pack/ruby.rb

```diff
@@ -799,7 +799,7 @@ def pg_adapters
   # @note execjs will blow up if no JS RUNTIME is detected and is loaded.
   # @return [Array] the node.js binary path if we need it or an empty Array
   def add_node_js_binary
-    bundler.has_gem?('execjs') && node_not_preinstalled? ? [@node_installer.binary_path] : []
+    bundler.has_gem?('execjs') || bundler.has_gem?('webpacker') && node_not_preinstalled? ? [@node_installer.binary_path] : []
```

Comment:
Move this to be multiple lines, having a ternary operator with logic is not ideal

```
if bundler.has_gem?('execjs') || bundler.has_gem?('webpacker') && node_not_preinstalled? ?
  [@node_installer.binary_path]
else
  []
end
```